src/commands/CommandSample.disable

```diff
@@ -0,0 +1,30 @@
+import { Command } from "../Command";
+import { CommandInteraction } from "discord.js";
+import { SlashCommandBuilder } from "@discordjs/builders";
+
+// set the class name, the export at the bottom and the file name to your desired command name
+class CommandName extends Command {
+    constructor() {
+        super("bunny"); // the name under which the bot internally stores your command
+    }
+
+    async execute(interaction: CommandInteraction): Promise<void> {
+        // put the logic of your command here
+        // for example:
+        const num = interaction.options.getInteger("amount") || 1;
+        await interaction.reply("🐇".repeat(num));
+    }
+
+    register() {
+        // set the name and description shown in discord here (these are necessary)
+        // (name: string.length = 1-32, description string.length = 1-100)(both lowercase!)
+        // here you can also add things like options to your command
+        // see more under https://discord.js.org/#/docs/builders/stable/class/SlashCommandBuilder
+        return new SlashCommandBuilder()
+            .setName("bunny")
+            .setDescription("shows you a cute bunny")
+            .addIntegerOption(option => option.setName("amount").setDescription("number of bunnies").setRequired(false));
```

Comment:
Should probably set a max and min value here. min 1 max of 2000(?) would make sence

Reply:
True.